sycl/include/CL/sycl/detail/pi.h

```diff
@@ -751,56 +752,62 @@ using pi_plugin = _pi_plugin;
 // populate the PI Version it supports, update targets field and populate
 // PiFunctionTable with Supported APIs. The pointers are in a predetermined
 // order in pi.def file.
-pi_result piPluginInit(pi_plugin *plugin_info);
+__SYCL_EXPORT pi_result piPluginInit(pi_plugin *plugin_info);
```

Comment:
Why do you need both this and linker script?

Reply:
> Why do you need both this and linker script?

Because @smaslov-intel wanted to remove "__ini" and "__fini" symbols from the exported symbol list.
Without the linker script, these automatically-added common symbols are present in libpi_opencl.so.